communication/contributor-comms/automation/announcement-guidelines.md

```diff
@@ -0,0 +1,49 @@
+# K8s Announcement Bot User guide
+
+The source code for this bot is present in https://github.com/kubernetes-sigs/slack-infra/tree/main/slack-post-message and it is deployed using https://github.com/kubernetes/k8s.io/tree/main/apps/slack-infra
+
+## Prerequisites
+
+### Permissions
+
+Reach out to the 'sig-contribex-comms' channel in kubernetes slack for permissions or for posting something on your behalf.
+
+### Adding bot to a channel
+
+In order for the bot to send message to a channel, it needs to be part of the channel first and before you add it, you need to join the channel. Although we have it added in most of the channels, it still might be required to be added to a new channel. Follow the below steps for doing this
```

Comment:
```suggestion
To allow the bot to send messages to a channel, it must first be added to the channel. Before adding it, you need to make sure that the bot has already joined the channel. Although the bot is added to most of the channels, it might still be necessary to add it to a new channel. Follow the steps below to do so.
```

Reply:
done